specs/003-aggiorna-tutto-il/quickstart.md

```
@@ -0,0 +1,117 @@
+# Compile a .vn file with SSE/SSE2 optimizations enabled
+jsavrs -i input.vn --enable-simd
+
+# Verify SIMD usage with verbose output
+jsavrs -i input.vn --enable-simd --verbose
+```
+
+### Example Input
+The following .vn code will benefit from SIMD optimizations:
+
+```rust
+// Vector arithmetic operations that will be optimized
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)